QUEST_LV_0100.tsv

```diff
@@ -1938,14 +1938,14 @@ QUEST_LV_0100_20150317_001937	Well done.{nl}It took little long because there wa
 QUEST_LV_0100_20150317_001938	It's so cruel to be a traitor.{nl}He was a respected swordsman, but even the monument after his death was not in good status.
 QUEST_LV_0100_20150317_001939	He even casted magic on the monument after it was broken into pieces.{nl}It's so hard to find those pieces because of those monsters that were inflicted on that magic.
 QUEST_LV_0100_20150317_001940	Ah, That's the object I ordered.{nl}It finally came.
-QUEST_LV_0100_20150317_001941	Don't you want to know what to do with it?{nl}I will call the powerful spirit from Dumbil pond.
+QUEST_LV_0100_20150317_001941	$Don't you want to know what to do with it?{nl}I will call the powerful spirit from Dumbla Pond.
 QUEST_LV_0100_20150317_001942	Thanks.{nl}It will take some time to dry it.{nl}
-QUEST_LV_0100_20150317_001943	It will be a plesant experience.
-QUEST_LV_0100_20150317_001944	I need a live creature to sacrifice to complete this spell.{nl}Lizard Man on the way to Dumbal Pond would be good.
-QUEST_LV_0100_20150317_001945	The spirits are mischievous and primitive.{nl}Lizard Man will be good for the experiement since it is very fast.
-QUEST_LV_0100_20150317_001946	The Lizard Man that you sent to us was really lively.{nl}Now, we are almost done.
-QUEST_LV_0100_20150317_001947	The peak of this spell will be Incroberk.{nl}I will give you the stone to control minds so please bring me Incroberk.
-QUEST_LV_0100_20150317_001948	Those Incroberks that you brought me are not enough.{nl}Please try harder.
+QUEST_LV_0100_20150317_001943	It will be a pleasant experience.
+QUEST_LV_0100_20150317_001944	$I need a live creature to sacrifice to complete this spell.{nl}Lizardmen on the way to Dumbla Pond would work.
+QUEST_LV_0100_20150317_001945	$The spirits are mischievous and primitive.{nl}Lizardmen are good for the experiment since they are very fast.
+QUEST_LV_0100_20150317_001946	$The Lizardmen that you sent to us were really lively.{nl}Now, we are almost done.
+QUEST_LV_0100_20150317_001947	$At the center of the spell I'm making are Infroburks.{nl}I will give you a stone that controls their minds, please bring me some Infroburk.
```

Comment:
the original korean sentence says "infroburks are the top of the spell" so I could only make out this kind of meaning